src/main/java/com/adit/backend/domain/place/service/command/UserPlaceCommandService.java

```diff
@@ -94,8 +98,8 @@ private void saveUserPlaceRelation(User user, CommonPlace commonPlace, UserPlace
 		notificationGenerationService.createNotificationOfASavedPlace(user, commonPlace, userPlace);
 	}
 
-	public boolean duplicatePlace(Long userId, PlaceRequestDto request) {
-		return userPlaceRepository.findDuplicatePlace(userId, request.url()) == null;
+	public boolean duplicatePlace(Long userId, String requestUrl) {
```

Comment:
```suggestion
	public boolean duplicatePlace(Long userId, String requestUrl) {
	   if(userPlaceRepository.findDuplicatePlace(userId, requestUrl)){
	   throw new PlaceException(USER_PLACE_DUPLICATE);
	   }
	}
```
위 형식으로 형식으로 검증후 바로 예외 처리를 진행하면 더 깔끔할것 같습니다.